src/screens/Events/EventDetails.js

```diff
@@ -627,12 +627,13 @@ const styles = {
 	}
 };
 
-const mapStateToProps = ({ events, user, members }) => {
+const mapStateToProps = ({ events, user, members, committees }) => {
```

Comment:
Don't know why committees was added back in, CommitteesList is not being used anywhere in here.